tests/integration_test/firebase_auth/firebase_auth_instance_e2e_test.dart

```diff
@@ -16,854 +16,867 @@ import './test_utils.dart';
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
-  group('FirebaseAuth.instance', () {
-    Future<void> commonSuccessCallback(currentUserCredential) async {
-      var currentUser = currentUserCredential.user;
-
-      expect(currentUser, isInstanceOf<Object>());
-      expect(currentUser.uid, isInstanceOf<String>());
-      expect(currentUser.email, equals(testEmail));
-      expect(currentUser.isAnonymous, isFalse);
-      expect(currentUser.uid, equals(FirebaseAuth.instance.currentUser!.uid));
-
-      var additionalUserInfo = currentUserCredential.additionalUserInfo;
-      expect(additionalUserInfo, isInstanceOf<Object>());
-      expect(additionalUserInfo.isNewUser, isFalse);
-
-      await FirebaseAuth.instance.signOut();
-    }
-
-    group('authStateChanges()', () {
-      StreamSubscription? subscription;
-      StreamSubscription? subscription2;
-
-      tearDown(() async {
-        await subscription?.cancel();
-        await ensureSignedOut();
-
-        if (subscription2 != null) {
-          await Future.delayed(const Duration(seconds: 5));
-          await subscription2.cancel();
-        }
-      });
-
-      test('calls callback with the current user and when auth state changes',
-          () async {
-        await ensureSignedIn(testEmail);
-        String uid = FirebaseAuth.instance.currentUser!.uid;
-
-        Stream<User?> stream = FirebaseAuth.instance.authStateChanges();
-        int call = 0;
-
-        subscription = stream.listen(
-          expectAsync1(
-            (User? user) {
-              call++;
-              if (call == 1) {
-                expect(user!.uid, isA<String>());
-                expect(user.uid, equals(uid)); // initial user
-              } else if (call == 2) {
-                expect(user, isNull); // logged out
-              } else if (call == 3) {
-                expect(user!.uid, isA<String>());
-                expect(user.uid != uid, isTrue); // anonymous user
-              } else {
-                fail('Should not have been called');
-              }
-            },
-            count: 3,
-            reason: 'Stream should only have been called 3 times',
-          ),
-        );
+  group(
```

Comment:
To save reviewers time, the only changes I've made to auth tests is to skip failing tests because of keychain sharing requirement:

```dart
    // macOS skipped because it needs keychain sharing entitlement. See: https://github.com/firebase/flutterfire/issues/9538
    skip: defaultTargetPlatform == TargetPlatform.macOS,
```

Reply:
This file was previously unformatted which is why there is a massive diff.